.ci.yaml

```diff
@@ -1076,6 +1076,37 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Linux android_preview_tool_integration_tests_1_1
```

Comment:
As discusssed offline, I think it should be sufficient to just test on linux, and bonus points if you can get rid of the 1_1 in the name and subshard field, but that's definitely not worth refactoring our test runner for.